src/main/resources/db/migration/all/2024100210000000__remove_email_fields_from_ap_areas.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ap_areas
```

Comment:
OK to remove these fields from the table and entity in this PR?  Leaving them in seems to cause not null violation errors.

Reply:
i would prefer we did this as a tidy up task because it makes rollbacks more difficult if we remove these columns, but i appreciate unstitching it now could be a headache so i'll leave it to you to decide

you're getting not null errors because you've removed them from `ApAreEntity` and the test code populating those fields (`ApAreaEntityFactory.kt`,`GivenAnApArea.kt`). Therefore, during tests we're no longer populating these non-null fields, leading to errors